src/test/java/com/padmaja/stringreplace/StringConversionTest.java

```diff
@@ -0,0 +1,26 @@
+package com.padmaja.stringreplace;
+
+import org.junit.Test;
+import java.lang.String;
+
+import static org.junit.Assert.assertEquals;
+
+//import static com.padmaja.stringreplace.stringReplace.manipulateString;
+//import static org.junit.Assert.*;
+
+ public class StringConversionTest {
+
+     @Test
+     public void replaceTest1() {
+
+        StringConversion util = new StringConversion();
+         //manipulateString("HelloWorld");
+         String result = util.manipulateString("HelloWorld");
+         //System.out.println(result);
+         String expectedresult = "vbppvzvbpz";
+         assertEquals(expectedresult,result);
+     }
+ }
+
```

Comment:
write a seprate test case for toupper.
all function has sepater test case.
like "aaa" to "AAA"
and "abc" to "ABC"